### PR TITLE
.*/: fix invalid relay dir output, delete Error Type for status

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
Current log of "no valid sub directory under relay dir" is not easy for users to understand. The `ErrorType` `UnknownError` will also confuse users.

### What is changed and how it works?
1. Refine log.
2. Delete `ErrorType` because we already have terror now.

### Check List 

Tests 

 - Unit test
 - Integration test

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: _utils/terror_gen/errors_release.txt [hunk 1]
```diff
@@ -159,13 +159,13 @@ ErrTaskCheckSameTableName,[code=26001:class=task-check:scope=internal:level=medi
...
-ErrTaskCheckSyncConfigError,[code=26005:class=task-check:scope=internal:level=medium],"%s %v: %v\n detail: %v"
+ErrTaskCheckSyncConfigError,[code=26005:class=task-check:scope=internal:level=medium],"%s: %v\n detail: %v"
 ErrTaskCheckGenBWList,[code=26006:class=task-check:scope=internal:level=medium],"generate black white list error"
 ErrRelayParseUUIDIndex,[code=28001:class=relay-event-lib:scope=internal:level=high],"parse server-uuid.index"
 ErrRelayParseUUIDSuffix,[code=28002:class=relay-event-lib:scope=internal:level=high],"UUID (with suffix) %s not valid"
 ErrRelayUUIDWithSuffixNotFound,[code=28003:class=relay-event-lib:scope=internal:level=high],"no UUID (with suffix) matched %s found in %s, all UUIDs are %v"
 ErrRelayGenFakeRotateEvent,[code=28004:class=relay-event-lib:scope=internal:level=high],"generate fake rotate event"
-ErrRelayNoValidRelaySubDir,[code=28005:class=relay-event-lib:scope=internal:level=high],"no valid relay sub directory exists"
+ErrRelayNoValidRelaySubDir,[code=28005:class=relay-event-lib:scope=internal:level=high],"there is no relay data under relay directory %s. please check relay data and status"
```
Comment (GregoryIan) [inline _utils/terror_gen/errors_release.txt]:
```suggestion
ErrRelayNoValidRelaySubDir,[code=28005:class=relay-event-lib:scope=internal:level=high],"there aren't any data under relay log directory %s. please check relay log using  query-status"
```

is it better?

**Comment by GregoryIan** [comment]
Cool! Rest LGTM.

File: _utils/terror_gen/errors_release.txt (hunk 1 above)
Comment (lichunzhu) [reply _utils/terror_gen/errors_release.txt]: Yes, addressed in https://github.com/pingcap/dm/pull/510/commits/bae91a62f440e7a362fc2f39ffd091206316726e

File: pkg/terror/error_list.go [hunk 2]
```diff
@@ -723,15 +723,15 @@ var (
...
 	ErrTaskCheckGenBWList        = New(codeTaskCheckGenBWList, ClassTaskCheck, ScopeInternal, LevelMedium, "generate black white list error")
 
 	// Relay log basic API error
 	ErrRelayParseUUIDIndex         = New(codeRelayParseUUIDIndex, ClassRelayEventLib, ScopeInternal, LevelHigh, "parse server-uuid.index")
 	ErrRelayParseUUIDSuffix        = New(codeRelayParseUUIDSuffix, ClassRelayEventLib, ScopeInternal, LevelHigh, "UUID (with suffix) %s not valid")
 	ErrRelayUUIDWithSuffixNotFound = New(codeRelayUUIDWithSuffixNotFound, ClassRelayEventLib, ScopeInternal, LevelHigh, "no UUID (with suffix) matched %s found in %s, all UUIDs are %v")
 	ErrRelayGenFakeRotateEvent     = New(codeRelayGenFakeRotateEvent, ClassRelayEventLib, ScopeInternal, LevelHigh, "generate fake rotate event")
-	ErrRelayNoValidRelaySubDir     = New(codeRelayNoValidRelaySubDir, ClassRelayEventLib, ScopeInternal, LevelHigh, "no valid relay sub directory exists")
+	ErrRelayNoValidRelaySubDir     = New(codeRelayNoValidRelaySubDir, ClassRelayEventLib, ScopeInternal, LevelHigh, "there aren't any data under relay log directory %s. please check relay log using  query-status")
```
Comment (csuzhangxc) [inline pkg/terror/error_list.go]:
```suggestion
	ErrRelayNoValidRelaySubDir     = New(codeRelayNoValidRelaySubDir, ClassRelayEventLib, ScopeInternal, LevelHigh, "there aren't any data under relay log directory %s. please check relay log using query-status")
```

**Pull request merged**